API/Data/app.config

```diff
@@ -24,7 +31,8 @@
     </defaultConnectionFactory>
     <providers>
       <provider invariantName="System.Data.SqlClient" type="System.Data.Entity.SqlServer.SqlProviderServices, EntityFramework.SqlServer" />
```

Comment:
This can be removed if we only rely on MySQL. However since we are considering to move back to mssql, let's take the decision on that.